src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,148 @@
+import { Registration, UnpublishPublicationRequest } from '../../../types/registration.types';
+import { Box, Breadcrumbs, Button, DialogActions, Divider, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+import { Modal } from '../../../components/Modal';
+import { ErrorMessage, Field, FieldProps, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { FindRegistration } from './FindRegistration';
+import { unpublishRegistration } from '../../../api/registrationApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { useDispatch } from 'react-redux';
+import { LoadingButton } from '@mui/lab';
+import { RequiredDescription } from '../../../components/RequiredDescription';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useHistory } from 'react-router-dom';
+import i18n from '../../../translations/i18n';
+import { useMutation } from '@tanstack/react-query';
+
+interface DeleteForm {
+  deleteMessage: string;
+}
+
+interface DeletePublicationProps {
+  registration: Registration;
+}
+
+export const DeletePublication = ({ registration }: DeletePublicationProps) => {
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [selectedDuplicate, setSelectedDuplicate] = useState<Registration | null>(null);
+  const history = useHistory();
+
+  const deleteValidationSchema = Yup.object().shape({
+    deleteMessage: Yup.string()
+      .min(3, t('feedback.validation.must_be_bigger_than', { field: i18n.t('common.justification'), limit: 3 }))
+      .required(t('feedback.validation.is_required', { field: i18n.t('common.justification') })),
+  });
+
+  const unpublishRegistrationMutation = useMutation({
+    mutationFn: (unpublishRequest: UnpublishPublicationRequest) =>
+      unpublishRegistration(registration.identifier, unpublishRequest),
+    onSuccess: () => {
+      setShowDeleteModal(false);
+      history.push(`/registration/${registration.identifier}?shouldNotRedirect`);
+    },
+    onError: () => {
+      dispatch(setNotification({ message: t('feedback.error.update_registration'), variant: 'error' }));
+    },
+  });
+
+  const handleDelete = async (values: DeleteForm) => {
+    const unpublishRequest = (): UnpublishPublicationRequest => {
+      return selectedDuplicate
+        ? {
+            type: 'UnpublishPublicationRequest',
+            duplicateOf: selectedDuplicate.id,
+            comment: values.deleteMessage,
+          }
+        : { type: 'UnpublishPublicationRequest', comment: values.deleteMessage };
+    };
+    unpublishRegistrationMutation.mutate(unpublishRequest());
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+        <Divider />
+        <Breadcrumbs
```

Comment:
`Breadcrumbs` ble kanskje brukt primært for å få inn `...`-knappen?
Breadcrumbs er en navigasjonskomponent, som egentlig skal brukes for å vise hvor brukeren er, og gi en rask vei til å hoppe tilbake til tidligere "parent"-visninger, typ `Min side / Profil / Kontaktinformasjon`. Derfor legger den automatisk inn `nav`, `ol`/`li` og noen `aria-label`, for å hjelpe brukeren med navigasjon, men i vårt tilfelle blir det bare misvisende.
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/ef64361b-6ee5-4297-b6c2-fb77e4248f4c)

Du kan se på `AssigneeSelector.tsx` for et eksempel på hvordan vi har løst det med en lignende knapp der:
```
<IconButton
  title="Min knapp"
  onClick={onClick}>
  <MoreHorizIcon />
</IconButton>
```